version.txt

```diff
@@ -1 +1 @@
-6.1.0rc1
+6.1.0rc2
```

Comment:
There is no https://dist.plone.org/release/6.1.0rc2 and we should really keep the image version matching there. I didn't follow why you used rc2 here?

Reply:
I changed the version in version.txt and at some other places temporaily to rc2 as a test to check if I was dealing with caching issues and/or pulling in an already existing rc1 server-builder or server-prod-config from hub.docker.com.  

My local builds of plone-backend were broken with a mixed Python3.11/Python3.12 environment that didn't make sense.   I'll update the PR, remove the older ones from hub.docker.com and see if we can release the images now properly as rc1 with GHA